app/javascript/v3/views/onboarding/SetupCompany.vue

```diff
@@ -0,0 +1,223 @@
+      </form-select>
+      <form-radio-tags
+        v-model="industry"
+        name="industry"
+        :label="$t('START_ONBOARDING.COMPANY.INDUSTRY.LABEL')"
+        :placeholder="$t('START_ONBOARDING.COMPANY.SIZE.PLACEHOLDER')"
+        :options="industryOptions"
+      />
+      <form-radio-tags
+        v-model="companySize"
+        name="companySize"
+        :label="$t('START_ONBOARDING.COMPANY.SIZE.LABEL')"
+        :placeholder="$t('START_ONBOARDING.COMPANY.SIZE.PLACEHOLDER')"
+        :options="companySizeOptions"
+      />
+    </div>
+    <submit-button
+      button-class="flex justify-center w-full text-sm text-center"
+      :button-text="$t('START_ONBOARDING.COMPANY.SUBMIT.BUTTON')"
+      @click="onSubmit"
+    />
+  </onboarding-base-modal>
+</template>
+
+<script>
+import FormInput from 'v3/components/Form/Input.vue';
+import FormSelect from 'v3/components/Form/Select.vue';
+import FormRadioTags from 'v3/components/Form/RadioTags.vue';
+import OnboardingBaseModal from 'v3/views/onboarding/BaseModal.vue';
+import { required, minLength } from 'vuelidate/lib/validators';
+import { mapGetters } from 'vuex';
+
+import SubmitButton from 'dashboard/components/buttons/FormSubmitButton.vue';
+import { timeZoneOptions } from 'dashboard/routes/dashboard/settings/inbox/helpers/businessHour.js';
+import {
+  getBrowserLocale,
+  getIANATimezoneFromOffset,
+} from 'v3/helpers/BrowserHelper';
+import {
+  findMatchingOption,
+  findIndustryOptions,
+  companySizeOptions,
+  industryOptions,
+} from 'v3/helpers/OnboardingHelper';
+import alertMixin from 'shared/mixins/alertMixin';
+import configMixin from 'shared/mixins/configMixin';
+export default {
+  components: {
+    FormInput,
+    SubmitButton,
+    FormSelect,
+    FormRadioTags,
+    OnboardingBaseModal,
+  },
+  mixins: [configMixin, alertMixin],
+  data() {
+    return {
+      companyName: '',
+      timezone: '',
+      locale: '',
+      companySize: '',
+      industry: '',
+      companySizeOptions,
+      industryOptions,
+    };
+  },
+  validations: {
+    companyName: {
+      required,
+      minLength: minLength(2),
+    },
+  },
+  computed: {
+    ...mapGetters({
+      getAccount: 'accounts/getAccount',
+      currentAccountId: 'getCurrentAccountId',
+    }),
+    accountDetails() {
+      return this.getAccount(this.currentAccountId);
+    },
+    languagesSortedByCode() {
+      const enabledLanguages = [...this.enabledLanguages];
+      return enabledLanguages.sort((l1, l2) =>
+        l1.iso_639_1_code.localeCompare(l2.iso_639_1_code)
+      );
+    },
+    timeZones() {
+      return [...timeZoneOptions()];
+    },
+    accountAttributes() {
+      const attributes = this.accountDetails.custom_attributes;
+      return {
+        companyName: this.accountDetails.name,
+        locale: this.accountDetails.locale,
+        companySize: attributes.company_size,
+        industry: attributes.industry,
+        timezone: attributes.timezone,
+      };
+    },
```

Comment:
Use destructuring to extract data from the accountDetails
```suggestion
    accountAttributes() {
      const { name: companyName, locale } = this.accountDetails;
      const {
        company_size: companySize,
        industry,
        timezone,
      } = this.accountDetails?.custom_attributes;
      return { companyName, locale, companySize, industry, timezone };
    },
```